serving-sdk/src/test/java/software/amazon/ai/mms/servingsdk/ModelServerEndpointTest.java

```diff
@@ -13,39 +13,186 @@
 
 package software.amazon.ai.mms.servingsdk;
 
-import junit.framework.Test;
-import junit.framework.TestCase;
-import junit.framework.TestSuite;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+import org.mockito.stubbing.Answer;
+import software.amazon.ai.mms.servingsdk.annotations.Endpoint;
+import software.amazon.ai.mms.servingsdk.annotations.helpers.EndpointTypes;
+import software.amazon.ai.mms.servingsdk.http.Request;
+import software.amazon.ai.mms.servingsdk.http.Response;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Properties;
 
 /**
  * Unit test for simple App.
  */
-public class ModelServerEndpointTest
-    extends TestCase
-{
-    /**
-     * Create the test case
-     *
-     * @param testName name of the test case
-     */
-    public ModelServerEndpointTest(String testName )
-    {
-        super( testName );
+public class ModelServerEndpointTest {
+    Context c;
+    Model m;
+    Worker w;
+    ModelServerEndpoint mse;
+    Request req;
+    Response rsp;
+    ByteArrayOutputStream outputStream;
+    Endpoint ea;
+
+    @Before
+    public void beforeSuite() throws IOException {
+        c = Mockito.mock(Context.class);
+        m = Mockito.mock(Model.class);
+        w = Mockito.mock(Worker.class);
+        ea = Mockito.mock(Endpoint.class);
+        mse = Mockito.mock(ModelServerEndpoint.class);
+        req = Mockito.mock(Request.class);
+        rsp = Mockito.mock(Response.class);
+        outputStream = new ByteArrayOutputStream();
+
+        Properties p = new Properties();
+        HashMap<String, Model> map = new HashMap<>();
+        List<Worker> l = new ArrayList<>();
+        map.put("squeezenet", m);
+        p.setProperty("Hello", "World");
+        c.getConfig();
+        l.add(w);
+
+        Mockito.when(c.getConfig()).thenReturn(p);
+        Mockito.when(c.getModels()).thenReturn(map);
+        Mockito.when(m.getModelWorkers()).thenReturn(l);
+        Mockito.when(m.getModelHandler()).thenReturn("mxnet_service:handle");
+        Mockito.when(m.getModelUrl()).thenReturn("/tmp/model/squeezenet.mar");
+        Mockito.when(m.getModelName()).thenReturn("squeezenet");
+        Mockito.when(w.getWorkerMemory()).thenReturn((long)100);
+        Mockito.when(w.isRunning()).thenReturn(false);
+        Mockito.when(ea.urlPattern()).thenReturn("myEndpoint");
+        Mockito.when(ea.description()).thenReturn("This is a test endpoint");
+        Mockito.when(ea.endpointType()).thenReturn(EndpointTypes.INFERENCE);
+        Mockito.when(rsp.getOutputStream()).thenReturn(outputStream);
+
```

Comment:
Did you format the code?

Reply:
Hmm. Today we don't have a auto formatter for maven. But, the checker that we have with maven doesn't seem to throw any warnings.